src/hooks/core/websocket/types.d.ts

```diff
@@ -110,13 +112,15 @@ export type GetstreamWebsocket = {
   type: string;
   unread_channels: number;
   unread_count: number;
-  targetName: string;
-  targetImage?: string;
   isAnonymous?: boolean;
   anon_user_info_emoji_name?: string;
   anon_user_info_color_name?: string;
   anon_user_info_emoji_code?: string;
   anon_user_info_color_code?: string;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

These newly added properties related to anonymous user information are optional and can be undefined. If these properties are not provided, make sure your code handles these cases gracefully without causing any runtime errors.



<!-- This is an auto-generated comment by OSS CodeRabbit -->